tests/test_graphical_units/test_creation.py

```diff
@@ -26,6 +28,27 @@ def test_DrawBorderThenFill(scene):
 
 
 # NOTE : Here should be the Write Test. But for some reasons it appears that this function is untestable (see issue #157)
+@pytest.mark.skipif(
+    not sys.platform.startswith("darwin"),
+    reason="Need different data for Windows and Linux",
+)
+@frames_comparison(last_frame=False)
+def test_Write(scene):
+    code = """from manim import Scene, Square
+
+    class FadeInSquare(Scene):
+        def construct(self):
+            s = Square()
+            self.play(FadeIn(s))
+            self.play(s.animate.scale(2))
+            self.wait()
+    """
+    rendered_code = Code(
+        code=code, tab_width=4, background="window", language="Python", font="Monospace"
+    )
+    scene.play(Write(rendered_code))
+
+
```

Comment:
I don't think we currently support having different control data for different OSs -- I'd remove this to get it merged so that we can include it in the bugfix release, and come back to implementing this consistently later.

```suggestion
```

Reply:
I thought having something would be better than nothing. The way I thought we could add tests for multiple platforms would be something like
```py
@pytest.mark.skipif( not sys.platform.startswith("darwin") )
@frames_comparison(last_frame=False)
def test_Write_MacOS(scene):
    ...

@pytest.mark.skipif( not sys.platform.startswith("linux") )
@frames_comparison(last_frame=False)
def test_Write_Linux(scene):
    ...

@pytest.mark.skipif( not sys.platform.startswith("win32") )
@frames_comparison(last_frame=False)
def test_Write_Windows(scene):
    ...
```
Well, maybe there is a cleaner way I suppose

Reply:
Requiring test data from different operating systems is not really practical. While you are right, having some sort of test would be better than having none, OS-dependent tests are rather painful to maintain: if a PR changes the test data, it would need to be regenerated on three different systems, stalling development indefinitely.

And as the fonts depend on the locally available / set default fonts, I'm not even sure whether the generated data really would be the same, globally, for all runs on on particular OS family.

Reply:
I agree. That does seem unnecessarily painful.
Fine then, this test can go until later.